### DISCUSSION TITLE
FIMS Filosophy Manuscript main message

### DISCUSSION THREAD

**Discussion by JaneSullivan-NOAA** [NOAA-FIMS/FIMS#456] [Ideas]
**What is our proposed main message for the FIMS manuscript?**
FIMS is a vehicle to implement best practices from multiple disciplines, including  fisheries and computer science.

**What are the "best practices" we are trying to implement?** 
-  Collaborative open science 
- Communication
- Modular and extensible architecture
- Well-tested and well-documented code

Please share your feedback with us on the main message and our draft list of best practices!

**Comment by k-doering-NOAA** [comment]
A little shorter: FIMS implements best practices from multiple disciplines, including fisheries and computer science.

**Comment by ChristineStawitz-NOAA** [comment]
I put this as the working title based on our "turning ideas into actions" session..thoughts??

Applying good software practices to developing a next-gen stock assessment platform: lessons learned building FIMS

**Reply by JaneSullivan-NOAA** [reply]
I like it! How's it going for you incorporating manuscript edits and trimming text?

**Comment by kellijohnson-NOAA** [comment]
The main message ended up being

> The Fisheries Integrated Modeling System uses collaborative software development to increase transparency, consistency, and pathways for uptake of good practices in fisheries stock assessment models.

Closing this discussion now that the manuscript is submitted to RPTS.
